src/complex/Pipeline/Messaging/OutputRenamedMessage.cpp

```diff
@@ -0,0 +1,31 @@
+#include "complex/Pipeline/Messaging/OutputRenamedMessage.hpp"
+
+using namespace complex;
+
+OutputRenamedMessage::OutputRenamedMessage(AbstractPipelineNode* pipeline, const PipelineFilter::RenamedPath& renamedOutput)
+: AbstractPipelineMessage(pipeline)
+, m_RenamedPath(renamedOutput)
+{
+}
+
+OutputRenamedMessage::~OutputRenamedMessage() = default;
```

Comment:
```suggestion
OutputRenamedMessage::~OutputRenamedMessage() noexcept = default;
```
Fixes exception specification mismatch.